package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -600,7 +600,7 @@ def _read_frame(self, frame):
             else:
                 raise NoDataError("Provide at least a position, velocity"
                                   " or force group in the h5md file.")
-        except ValueError:
+        except IndexError:
```

Comment:
Let's make this 2.0 and 3.0 compatible:
```suggestion
        except (ValueError, IndexError):
```